tasks/section4.yml

```diff
@@ -390,6 +453,29 @@
       - patch
       - rule_4.2.1.3
 
+- name: "SCORED | 4.2.1.3 | PATCH | Ensure rsyslog default file permissions configured"
+  file:
+      path: "{{ item }}"
+      mode: 0600
+      state: touch
+      owner: root
+      group: root
+      modification_time: "preserve"
```

Comment:
Likely will have to bump minimum version of ansible to use this?

Reply:
Will do that instead with if/else and re-commit.  I didn't think about version dependency.